airbyte-scheduler/src/main/java/io/airbyte/scheduler/persistence/DefaultJobPersistence.java

```diff
@@ -420,7 +420,8 @@ private Job getJob(DSLContext ctx, long jobId) {
         .collect(Collectors.toList());
   }
 
-  public static Optional<Job> getJobFromResult(Result<Record> result) {
+  @VisibleForTesting
+  static Optional<Job> getJobFromResult(Result<Record> result) {
```

Comment:
this seemed like it was only used in a private method and was exposed only for testing purposes.